docs/source/models/cause.rst

```diff
@@ -146,26 +146,107 @@ What does a model document look like?
++---------+--------------------------------+---------------------------------------------------------------------+
+|Neonatal_|With and without condition      |Simulants are born either with or without condition                  |
+|         |                                |                                                                     |
++---------+--------------------------------+---------------------------------------------------------------------+
+
+.. _SI:
 
 SI
 ++
 
+.. image:: SI.png
+
+In this cause model structure, simulants in the susceptible state can transition to the infected state, where they 
+will remain for the remainder of the simulation. 
+
+This cause model structure is appropriate for chronic conditions from which individuals can never recover.
+
+Examples of conditions appropriate for SI cause models include Alzheimer’s disease and other dementias.
+
+.. _SIS:
+
 SIS
 +++
 
+.. image:: SIS.png
+
+In this cause model structure, simulants in the susceptible state can transition to the infected state and 
+simulants in the infected state can transition to the susceptible state. Notably, this cause model allows for
+simulants to enter the infected state more than once in a simulation. 
+
+This cause model structure is appropriate for conditions for which individuals can have multiple cases over 
+their lifetimes.
+
+Examples of conditions appropriate for SIS cause models include diarrheal diseases.
+
+.. _SIR:
+
 SIR
 +++
 
+.. image:: SIR.png
+
+In this cause model structure, simulants in the susceptible state can transition to the infected state and 
+simulants in the infected state can transition to a recovered state where they will remain for the remainder
+of the simulation. Notably, the cause model allows individuals to become infected only once in a simulation.
+
+This cause model structure is appropriate for conditions for which individuals can only have a single case, but 
+do not stay in the with condition state forever.
+
+An example of a condition appropriate for SIR cause models is measles.
+
+.. _Neonatal:
+
 Neonatal
 ++++++++
 
+In this cause model structure, simulants who are born during the simulation can be born either with condition 
+or without condition. Whether or not simulants born with condition may recover from the with condition state 
+depends on the cause at hand.
+
+This cause model structure is appropriate for conditions that are present at birth.
+
+An example of a condition appropriate for a neonatal cause model without recovery from the condition state is 
+preterm birth.
+
+.. image:: Neonatal_no_recover.png
+
+An example of a condition appropriate for a neonatal cause model with potential recovery from the condition 
+state is cleft palate (which may be treated later in life).
+
+.. image:: Neonatal_recover.png
+
+.. _`Non-Standard Cause Models`:
+
+Non-Standard Cause Models
```

Comment:
Perhaps "Non-Standard" is not the right terminology, because there isn't really a "Standard," but I'm not sure what a better term would be...

Reply:
I changed the headings to "Basic Cause Model Structures" and "Other Cause Model Structures" in the most recent commit